internal/output/print.go

```diff
@@ -9,6 +15,8 @@ func Print(data interface{}) (err error) {
 	switch globalOutput.format {
 	case FormatJSON:
 		err = globalOutput.json(data)
+	case FormatText:
+		err = globalOutput.table(data)
```

Comment:
`FormatText ` == `.table(data)`? This is probably a semantic thing, but "text" and "table" feel like different output types to me.

Reply:
That's totally a bug, should be `.text`, standby. (Was FormatTable previously)